packages/terra-tag/package.json

```
@@ -23,14 +23,18 @@
   "homepage": "https://github.com/cerner/terra-core#readme",
   "peerDependencies": {
     "react": "^16.8.5",
-    "react-dom": "^16.8.5"
+    "react-dom": "^16.8.5",
+    "react-intl": ">=2.8.0 <6.0.0"
   },
   "dependencies": {
     "classnames": "^2.2.5",
     "keycode-js": "^3.1.0",
     "prop-types": "^15.5.8",
-    "terra-icon": "^3.56.0",
-    "terra-theme-context": "^1.0.0"
+    "resize-observer-polyfill": "^1.4.1",
+    "terra-icon": "^3.54.0",
+    "terra-theme-context": "^1.0.0",
+    "terra-visually-hidden-text": "^2.36.0",
+    "uuid": "^9.0.0"
```

We are using uuid v3.4.0
```suggestion
    "uuid": "3.4.0"
```